src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -511,4 +535,36 @@ private void importEntries(List<CitationRelationItem> entriesToImport, CitationF
 
         dialogService.notify(Localization.lang("Number of entries successfully imported") + ": " + entriesToImport.size());
     }
+
+    /**
+     * Function to open possible duplicate entries window to compare duplicate entries
+     *
+     * @param duplicateItem duplicate in the citation relations tab
+     */
+    private void openPossibleDuplicateEntriesWindow(CitationRelationItem duplicateItem) {
+        BibEntry localEntry = duplicateItem.localEntry();
+        BibEntry duplicateEntry = duplicateItem.entry();
+
+        MergeEntriesDialog dialog = new MergeEntriesDialog(localEntry, duplicateEntry, preferences);
+        dialog.setTitle(Localization.lang("Possible duplicate entries"));
+
+        Optional<EntriesMergeResult> mergeResultOpt = dialogService.showCustomDialogAndWait(dialog);
+        mergeResultOpt.ifPresentOrElse(entriesMergeResult -> {
+            entriesMerge = entriesMergeResult.mergedEntry();
```

Comment:
Please rename to `mergedEntry`. -- The variable name should state what's stored inside. It are not multiple entries. Moreover, consistency with a method name is nice.